rero_ils/permissions.py

```diff
@@ -60,6 +60,22 @@ def staffer_is_authenticated(user=None):
     return None
 
 
+def patron_is_authenticated(user=None):
+    """Checks if user (with role patron) is authenticated.
```

Comment:
you can use sphinx documentation format: http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#field-lists